tests/unittests/graphGradTest.cpp

```diff
@@ -181,3 +181,30 @@ TEST(GraphAutoGrad, checkPlaceholderGradTest) {
   /// node will be added.
   EXPECT_GE(A->getNumUsers(), 1);
 }
+
+/// Check that we can differentiate functions that use ConvertToNode.
+TEST(GraphAutoGrad, checkConvertToGradTest) {
+  ExecutionEngine EE;
+  TrainingConfig TC;
+  Context ctx;
+
+  // Construct the network:
+  TC.learningRate = 0.001;
+
+  auto &mod = EE.getModule();
+  Function *F = mod.createFunction("main");
+
+  auto *A = mod.createPlaceholder(ElemKind::FloatTy, {20, 13}, "A", false);
+  auto inputHandle = ctx.allocate(A)->getHandle<float_t>();
```

Comment:
Why `float_t` instead of `float`?

Reply:
Good catch!
Editing issue :).
Switch `float16_t` into `float`, but miss the `_t`. I am actually surprised it compiled.

Reply:
Done